concepts/modularization/modularization_overview.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Modularization in Kyma
+---
+
+With Kyma’s modular approach, you can install exactly the Kyma components you want, making the installation very lightweight and adjusted to your business needs. Kyma provides three components to support the modular approach: The Module Manager, the Lifecycle Manager, and the Runtime Watcher.
+
+## Kyma’s Modular Approach
+
+![](assets/Modular_Approach.svg)
+
+## 1. Kyma Module
+
+A Kyma module contains all information required to install and run the associated components in a Kyma runtime:
+
+* A renderable Kubernetes manifest that includes the Kubernetes resources of the module
+* A default configuration for module chart resource installation
+* Optionally, further information
+
+All these assets are bundled into a single container image using the OCI image specification.
+
+## 2. Module Manager
+
+The Module Manager installs and manages the Kyma components in a local or remote setup. For example, it remotely manages and synchronizes the installation in the SAP Kyma Runtime. The Module Manager retrieves the container image and processes the Kubernetes manifest by calling a rendering framework like Helm or Kustomize. Finally, it deploys the manifest in Kubernetes and monitors the outcome of the deployment.
```

Comment:
```suggestion
The Module Manager installs, uninstalls and manages the Kyma modules in a local (single cluster mode) or remote setup. E.g., it remotely manages and synchronizes Kyma module manifest resources in the SAP Kyma Runtime. The Module Manager retrieves the image layers from the specified image registry and processes the manifest resources by calling a rendering framework like Helm or Kustomize. Finally, it deploys this resource along with a custom resource to track state changes. It ensures consistency both via a time-based reconciliation and watching on state changes of the deployed custom resource in Kyma Runtime.
```

Reply:
Just minor language editing to @adityabhatia's suggestion:

```suggestion
The Module Manager installs, uninstalls, and manages the Kyma modules in a local (single cluster mode) or remote setup. For example, it remotely manages and synchronizes Kyma module manifest resources in the SAP Kyma Runtime. The Module Manager retrieves the image layers from the specified image registry and processes the manifest resources by calling a rendering framework like Helm or Kustomize. Finally, it deploys this resource along with a custom resource to track state changes. It ensures consistency both with a time-based reconciliation and by watching state changes of the deployed custom resource in Kyma runtime.
```